test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -136,56 +137,58 @@ let paused = false;
 }
 
 {
-  let file7 =
+  let data = '';
+  let file =
     fs.createReadStream(rangeFile, Object.create({autoClose: false }));
-  assert.strictEqual(file7.autoClose, false);
-  file7.on('data', common.noop);
-  file7.on('end', common.mustCall(function() {
+  assert.strictEqual(file.autoClose, false);
+  file.on('data', (chunk) => { data += chunk; });
+  file.on('end', common.mustCall(function() {
     process.nextTick(common.mustCall(function() {
-      assert(!file7.closed);
-      assert(!file7.destroyed);
-      file7Next();
+      assert(!file.closed);
+      assert(!file.destroyed);
+      assert.strictEqual(data, 'xyz\n');
+      fileNext();
     }));
   }));
 
-  function file7Next() {
+  function fileNext() {
     // This will tell us if the fd is usable again or not.
-    file7 = fs.createReadStream(null, Object.create({fd: file7.fd, start: 0 }));
-    file7.data = '';
-    file7.on('data', function(data) {
-      file7.data += data;
+    file = fs.createReadStream(null, Object.create({fd: file.fd, start: 0 }));
```

Comment:
this is just weird, shadowing `file` but AFAICT only affects L147-8 , so there it asserts the state of this `file` 🤔 
IMHO it's wrong, but maybe not for this PR...